.github/workflows/build-matrix.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
 
                   # Packages (components and bundles)
                   PACKAGES="[]"
-                  for pkg in $(find src/ -mindepth 2 -type f -name composer.json -not -path "*/vendor/*" -not -path "*/Bridge/*" | xargs -I{} dirname {} | sed 's|^src/||' | grep -Ev "examples" | sort); do
+                  for pkg in $(find src/ -mindepth 2 -type f -name composer.json -not -path "*/vendor/*" -not -path "*/Bridge/*" -not -path "*/tests/*" | xargs -I{} dirname {} | sed 's|^src/||' | grep -Ev "examples" | sort); do
```

Comment:
this was neccesary so that my test directory was not picked to create a github-action